api/v1alpha1/zync_types.go

```diff
@@ -300,6 +320,35 @@ func (zrs *ZyncRailsSpec) Default() {
 	zrs.MaxThreads = intOrDefault(zrs.MaxThreads, pointer.Int32(zyncDefaultConfigRailsMaxThreads))
 }
 
+// ZyncRailsConsoleSpec configures the App component of System
+type ZyncRailsConsoleSpec struct {
+	// Enables or disables the Zync Console statefulset
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	Enabled *bool `json:"enabled,omitempty"` // Image specification for the Console component.
+	// Defaults to zync image if not defined.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	Image *ImageSpec `json:"image,omitempty"`
+	// Resource requirements for the component
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	Resources *ResourceRequirementsSpec `json:"resources,omitempty"`
+	// Describes node affinity scheduling rules for the pod.
+	// +optional
+	NodeAffinity *corev1.NodeAffinity `json:"nodeAffinity,omitempty" protobuf:"bytes,1,opt,name=nodeAffinity"`
+	// If specified, the pod's tolerations.
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty" protobuf:"bytes,22,opt,name=tolerations"`
+}
+
+// Default implements defaulting for the system App component
```

Comment:
wrong comment?

Reply:
fixed in https://github.com/3scale-ops/saas-operator/pull/263/commits/f7909daed91e94d2e466ff1371f807cad17fc5d2